src/Adapters/JobsAdapter.ts

```diff
@@ -0,0 +1,146 @@
+import ADTJobsData from '../Models/Classes/AdapterDataClasses/ADTJobsData';
+import AdapterMethodSandbox from '../Models/Classes/AdapterMethodSandbox';
+import {
+    AdapterMethodParamsForJobs,
+    ADT_ApiVersion,
+    AxiosObjParam,
+    IAuthService,
+    IJobsAdapter,
+    LOCAL_STORAGE_KEYS
+} from '../Models/Constants';
+import { applyMixins, validateExplorerOrigin } from '../Models/Services/Utils';
+import ADT3DSceneAdapter from './ADT3DSceneAdapter';
+import ADTAdapter from './ADTAdapter';
+
+export default class JobsAdapter implements IJobsAdapter {
```

Comment:
Agree, we need a ProxyOrCORSAdapater that JobsAdapter and AdtAdapter and BlobAdapter all extend

Reply:
The only thing that adapters could have in common would be the `useProxy` variable, they all may have to implement their own `generateHeaders` or `generateUrl` methods and the request building is done when the params are passed to Axios functions.

Reply:
Except that they all call ADT services so they will always need that same header and same URL right?

Reply:
Okay I just spoke to Pato and lets hold off on this because it is only common between Jobs and ADTAdapter.  If we broke ADTAdapter down further (say to have an adapter per set of APIs for ADT) it would make a lot more sense to have a shared implementation for switching to be able to use CORS.  For now, let's dupe it but keep note of the refactor for later (Pato will need it for twin mapping and Irem for Event Handlers)

Reply:
So the thinking is that he will do the refactor when they make the next round of granular adapters?

Reply:
Yep

Reply:
Matt did this already and push the changes into my branch